<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Res2.qrc">
    <normaloff>:/pix/Bild2.png</normaloff>
    <normalon>:/pix/Bild1.png</normalon>
    <activeoff>:/pix/Bild1.png</activeoff>:/pix/Bild2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QVideoWidget" name="videoWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Datei">
    <property name="title">
     <string>&amp;Datei</string>
    </property>
    <addaction name="action_ffnen"/>
    <addaction name="separator"/>
    <addaction name="action_Beenden"/>
   </widget>
   <widget class="QMenu" name="menu_Wiedergabe">
    <property name="title">
     <string>&amp;Wiedergabe</string>
    </property>
    <addaction name="action_Pause"/>
    <addaction name="action_Stopp"/>
   </widget>
   <widget class="QMenu" name="menu_Ansicht">
    <property name="title">
     <string>&amp;Ansicht</string>
    </property>
    <widget class="QMenu" name="menu_Symbolleisten">
     <property name="title">
      <string>&amp;Symbolleisten</string>
     </property>
     <addaction name="action_Lautst_rke"/>
     <addaction name="actionW_iedergabe"/>
    </widget>
    <addaction name="action_Wiedergabeliste"/>
    <addaction name="menu_Symbolleisten"/>
    <addaction name="separator"/>
    <addaction name="action_Vollbild"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menu_Datei"/>
   <addaction name="menu_Wiedergabe"/>
   <addaction name="menu_Ansicht"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBarWiedergabe">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_ffnen"/>
   <addaction name="separator"/>
   <addaction name="action_Pause"/>
   <addaction name="action_Stopp"/>
  </widget>
  <widget class="QToolBar" name="toolBarLautstaerke">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTon_ein_aus"/>
  </widget>
  <action name="action_Beenden">
   <property name="text">
    <string>&amp;Beenden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_ffnen">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/oeffnen.png</normaloff>:/images/oeffnen.png</iconset>
   </property>
   <property name="text">
    <string>Ö&amp;ffnen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Pause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/pause.png</normaloff>:/images/pause.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionW_iedergeben">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
   </property>
   <property name="text">
    <string>W&amp;iedergeben</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Stopp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stopp</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionTon_ein_aus">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/images/mute.png</normaloff>:/images/mute.png</iconset>
   </property>
   <property name="text">
    <string>Ton ein/aus</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;mute&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="action_Lautst_rke">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Lautstärke</string>
   </property>
  </action>
  <action name="actionW_iedergabe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>W&amp;iedergabe</string>
   </property>
  </action>
  <action name="action_Wiedergabeliste">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Wiedergabeliste</string>
   </property>
  </action>
  <action name="actionEintrag_l_schen">
   <property name="text">
    <string>Eintrag löschen</string>
   </property>
  </action>
  <action name="actionEintr_ge_sortieren">
   <property name="text">
    <string>Einträge sortieren</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info...</string>
   </property>
  </action>
  <action name="action_Vollbild">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Vollbild</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Res.qrc"/>
  <include location="Res2.qrc"/>
 </resources>
 <connections/>
</ui>
